robot_skills/src/robot_skills/arms.py

```diff
@@ -126,6 +119,12 @@ def send_joint_trajectory(self, configuration, timeout=5):
                        "Specify get_arm(..., required_trajectories=['{}'])".format(configuration))
         return self._arm.send_joint_trajectory(configuration, timeout)
 
+    def send_goal(self, frameStamped, timeout=30, pre_grasp=False, first_joint_pos_only=False,
```

Comment:
Shouldn't pre_grasp be 'true' by default?

Reply:
I don't know. Just matching the API of the hidden arm.